documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -860,7 +860,9 @@ Here is an example of a change event value in an event that the connector genera
         "ord": 1,
         "h": null,
         "tord": null,
-        "stxnid": null
+        "stxnid": null,
+        "lsid":"{\"id\": {\"$binary\": \"FA7YEzXgQXSX9OxmzllH2w==\",\"$type\": \"04\"},\"uid\": {\"$binary\": \"47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=\",\"$type\": \"00\"}}",
```

Comment:
Oh, wow, that's... huge? Do we really need this? And I think that's not the only example where this should be added?

Reply:
And should we mark the "change streams" capturing mode as incubating? Or are we good with releasing it as "stable"?

Reply:
@gunnarmorling This is present only when the change was done in trasnaction. Otherwise it is empty. That's the reason why I dedicated much space to it.

Re - incubating, I don't know. The problem is that it is must for MongoDB 5. So I am torn on this one.

Reply:
Ok, logged https://issues.redhat.com/browse/DBZ-4420 for revisiting this before the 1.8.0.Final release.